decidim-blogs/spec/forms/decidim/blogs/admin/post_form_spec.rb

```diff
@@ -52,15 +54,35 @@ module Admin
             subject.map_model(post)
           end
 
-          it "does not assign the user group for normal users" do
-            expect(subject.user_group_id).to be_nil
+          it "assigns current_user as user_group_id" do
+            expect(subject.user_group_id).to eq("current_user")
+          end
+
+          it "assigns user name as author" do
+            expect(subject.author).to be_nil
+          end
+
+          context "when author is an organization" do
+            let(:author) { current_organization }
+
+            it "assigns 'current_organization' as user_group_id" do
+              expect(subject.user_group_id).to eq("current_organization")
+            end
+
+            it "assigns current_organization as author" do
+              expect(subject.author).to eq(author)
+            end
           end
```

Comment:
```suggestion
          context "when author is an user" do
            let(:author_id) { "current_user" }

            it "assigns 'current_user' as user_group_id" do
              expect(subject.user_group_id).to eq("current_user
            end

            it "assigns current_user as author" do
              expect(subject.author).to eq(current_user)
            end
          end
```
In here (sorry I've started with organization then the user, but you get the logic). You need to test when the parameter for the form is "current_user", the `author` method returns the object user.